examples/Console/TestJaegerExporter.cs

```diff
@@ -13,6 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 // </copyright>
+using System.Collections.Generic;
```

Comment:
nit: consider adding a blank line between L15 and L16.

Reply:
We ended up reverting changes to this file but will add the blank line before using directives